lib/Malkist.rb

```diff
@@ -1,7 +1,12 @@
+# frozen_string_literal: true
+
 require 'Malkist/version'
+require 'Malkist/distances'
 
+# Class Malkist is a class for Google Maps Service APIs wrapper. It will wrap the response into a deliverable object.
 class Malkist
```

Comment:
kalau ga ngebuat instance (cuman class method), prefer makai `module` instead of `class`